prod/server.js

```diff
@@ -1,6 +1,6 @@
 import express from 'express'
 import React from 'react'
-import {renderToString} from 'react-dom/server'
+import {renderToString} from 'react/dist/react.min'
```

Comment:
ouch ouch

reverted in https://github.com/siddharthkp/reaqt/commit/47bfa4b8fa1c28aeb6bc0fee3a456a927e5798bb